resources/todomvc/vanilla-examples/javascript-web-components/src/index.js

```diff
@@ -0,0 +1 @@
+// placeholder
```

Comment:
Is it useful for something? I believe you can remove this file and from `index.html`, right?